compass/ocean/tests/soma/initial_state.py

```diff
@@ -0,0 +1,106 @@
+import xarray
+
+from mpas_tools.io import write_netcdf
+from mpas_tools.mesh.conversion import convert, cull
+
+from compass.step import Step
+from compass.model import run_model
+
+
+class InitialState(Step):
+    """
+    A step for creating a mesh and initial condition for SOMA test cases
+
+    Attributes
+    ----------
+    resolution : str
+        The resolution of the test case
+    """
+
+    def __init__(self, test_case, resolution):
+        """
+        Create the step
+
+        Parameters
+        ----------
+        test_case : compass.TestCase
+            The test case this step belongs to
+
+        resolution : str
+            The resolution of the test case
+        """
+        super().__init__(test_case=test_case, name='initial_state')
```

Comment:
This is implicitly defaulting to running on one core.  We may want to run on multiple cores, particularly at higher res.